src/main/java/CellContent.java

```diff
@@ -0,0 +1,97 @@
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellType;
+
+public class CellContent {
+	public CellType type;
+	public boolean bool;
+	public double numeric;
+	public String string;
+	
+	public boolean wasFormula;
+	
+	public CellContent(Cell cell) {
+		if(cell == null) {
+			type = CellType.BLANK;
+			return;
+		}
+		
+		CellType temp_type = cell.getCellTypeEnum();
+		
+		wasFormula = false;
+		if(temp_type == CellType.FORMULA) {
+			temp_type = cell.getCachedFormulaResultTypeEnum();
+			wasFormula = true;
+		}
+		
+		type = temp_type;
```

Comment:
Good catch, I think this is a leftover from an earlier version before I made a structural change.